### ISSUE TITLE
Feature Request: Add macro

### ISSUE THREAD

**Issue by electricalwind** [dukeboard/talk-app#4]
It could be nice to add macro handling to allow the user to customize the talk without having to retype the html code each type.
e.g: 
for this text:
```
Red is nice
```
I would like to put the red in red and with a specific font
I can do : 
```
<span style="color:red; font-family:Avenir; font-size:2em;">red.</span> is nice
```
but if I want to apply it again in 10 other places it could become a bit repetitive.
The possibility of creating a macro that allow me to apply a specific HTML tag to a selected text could improve a lot the productivity ;)

**Comment by dukeboard** [comment]
Good idea i think i will reuse HTML5 components for that, like being able to do instead <redSpan>myRedText</redSpan>
